app/screens/channel_info/channel_info.js

```diff
@@ -82,6 +82,7 @@ export default class ChannelInfo extends PureComponent {
             isFavorite: props.isFavorite,
             isMuted: props.isChannelMuted,
             ignoreChannelMentions: props.ignoreChannelMentions,
+            prevPropsUserId: props.currentUserId,
```

Comment:
why do you need the `currentUserId` in the state?

Reply:
Reason for keeping this here was to update `state` from `props` only when the `currentUserId` changes and in other cases `state` update would be handled by the `handler`. But, as you mentioned in the other comment, I think this approach is not correct